tests/TestTag.shadow-dom.test.js

```diff
@@ -9,11 +9,15 @@ import { syncRaf } from './test-utils.js';
 // TODO: Needs unit tests to validate attributes/props being handled correctly (at minimum pass through)
 
 let el = null;
+let cssFile = 'test.css';
+let cssFiles = ['test0.css', 'test1.css', 'test2.css'];
 
 describe('<test-tag> (Shadow DOM)', () => {
 
 	beforeAll(() => {
-		svelteRetag({ component: TestTag, tagname: 'test-shad', shadow: true });
+		svelteRetag({ component: TestTag, tagname: 'test-shad', shadow: true});
+		svelteRetag({ component: TestTag, tagname: 'test-shad-single-css', shadow: true, href: cssFile});
```

Comment:
You could theoretically move this `svelteRetag()` (and the corresponding muli-css) into their corresponding unit tests. Does that work? If so maybe let's go that way.